mmaction/models/backbones/c3d.py

```diff
@@ -0,0 +1,140 @@
+from mmcv.cnn import ConvModule, constant_init, kaiming_init, normal_init
+from mmcv.runner import load_checkpoint
+from mmcv.utils import _BatchNorm
+from torch import nn as nn
+
+from ...utils import get_root_logger
+from ..registry import BACKBONES
+
+
+@BACKBONES.register_module()
+class C3D(nn.Module):
+    """C3D backbone.
+
+    Args:
+        pretrained (str | None): Name of pretrained model.
+        style (str): ``pytorch`` or ``caffe``. If set to "pytorch", the
+            stride-two layer is the 3x3 conv layer, otherwise the stride-two
+            layer is the first 1x1 conv layer. Default: 'pytorch'.
+        conv_cfg (dict): Config dict for convolution layer.
+            Default: ``dict(type='Conv3d')``.
```

Comment:
Default: None

Reply:
```python
conv_cfg (dict | None): Config dict for convolution layer. If set to None, it will use ``dict(type='Conv3d')`` to construct layers. Default: None.
```